client/src/client.rs

```diff
@@ -687,7 +711,20 @@ impl<B, E, Block, RA> Client<B, E, Block, RA> where
 
 				operation.op.update_cache(new_cache);
 
-				let (main_sc, child_sc, tx, _, changes_trie_tx) = storage_changes.into_inner();
+				let (main_sc,
+					child_sc,
```

Comment:
Funky indentation here.

Reply:
I think this was my suggestion and it's generally the style I follow, except I keep it to only 1 field per line.